packages/web-console/src/partials/import.hbs

```diff
@@ -28,61 +28,51 @@
         <div id="import-top" class="ibox-content">
           <div id="dragTarget" class="drag-target drag-idle">
             <div>
-              <i class="fa fa-copy fa-3x" style="margin: 0 15px 20px 0;"></i>
+              <i class="fa fa-copy fa-3x" style="margin: 0 15px 20px 0"></i>
               <i
                 class="fa fa-folder-open fa-3x"
-                style="margin: 0 15px 20px 0;"
-              ></i>
-              <i
-                class="fa fa-file-alt fa-3x"
-                style="margin: 0 15px 20px 0;"
+                style="margin: 0 15px 20px 0"
               ></i>
+              <i class="fa fa-file-alt fa-3x" style="margin: 0 15px 20px 0"></i>
             </div>
-            <h2 style="font-weight: 500;">
-              Drag files or paste data here to
-              import
+            <h2 style="font-weight: 500">
+              Import CSV file
```

Comment:
Seeing the screenshot, I wonder if this line is needed at all. 
Should we consider changing this to "click here to browse" with the link?

Reply:
I actually tried exactly that, but it felt a bit empty, I feel like some explanation of a CTA makes sense.

Wdyt?

Reply:
![image](https://user-images.githubusercontent.com/4284659/185907105-77887247-cd15-419c-8045-f830ab6a1a2f.png)

Reply:
Actually you're right! I think the original one looks better!